common/config/config.go

```diff
@@ -100,8 +100,12 @@ type (
 		Algorithm string `yaml:"algorithm"`
 		// Public Key Path for verifying JWT token passed in from external clients
 		PublicKey string `yaml:"publicKey"`
+		// Public Key loaded once
+		PublicKeyLoaded []byte `yaml:"-"`
 		// Private Key Path for creating JWT token
 		PrivateKey string `yaml:"privateKey"`
+		// Private Key loaded once
+		PrivateKeyLoaded []byte `yaml:"-"`
```

Comment:
Let's not adding these two entries here. The loaded logic can be done outside to leave the config clean.